e2e_playwright/st_expander_test.py

```diff
@@ -50,35 +50,24 @@ def test_expander_displays_correctly(
 
 def test_expander_collapses_and_expands(app: Page):
     """Test that an expander collapses and expands."""
-    main_container = app.get_by_test_id("stMain")
-    main_expanders = main_container.get_by_test_id("stExpander")
-    expect(main_expanders).to_have_count(
-        NUMBER_OF_EXPANDERS - 1
-    )  # -1 to subtract sidebar
-
-    expanders = main_expanders.all()
-    # Starts expanded
-    expander_header = expanders[0].locator(EXPANDER_HEADER_IDENTIFIER)
-    expect(expander_header).to_be_visible()
-    toggle = expander_header.locator("svg").first
-    expect(toggle).to_be_visible()
+
+    # Check that content is initially visible (starts expanded)
+    expanded_expander = get_expander(app, "Normal expanded")
```

Comment:
I rewrote this test to be a bit more straightforward and check that the expander content shows up when expanding/collapsing, instead of just checking for the arrow icon.